README.md

```diff
@@ -1,7 +1,30 @@
+<img align="right" src="https://i.imgur.com/eE3GJgb.png" height="200" width="200">
+
 # The Brass Knights RPG Support Page
-Public Support Repo for BK RPG
+This is the public support repo for the Brass Knights RPG.
+
+Please use this repository to report bugs and request new features for the Brass Knights RPG.
+
+## Conduct
+There are very generic rules that should be followed by everyone at all times:
+- Do not insult or harass anyone.
+- Do not repedatly bump an open issue remarking that the issue still persists.
+- Do not use ALL CAPS when opening / creating or editing issues and comments. 
 
 ## How to file an issue:
-Go to the [Issues](https://github.com/Avalon-Entertainment/bkrpg-support/issues) tab and click New Issue.
+If you encounter a bug while playing the Brass Knights RPG or have a request, please look up wheter your report can be found in the [Issues](https://github.com/Avalon-Entertainment/bkrpg-support/issues) list.
+If not, please create a new issue with the following data, when you...
+
+... Report a bug:
+    - A short and descriptive title
+    - A description of the issue
+    - The reproduction steps
+
+... Request a new feature:
+    - A descriptive title
+    - A clear description on what you request
+    - (Optional) Justification on why you think it should be implemented
 
-Enter the title and a description and I'll try to get back to you ASAP.
+### Related Links
```

Comment:
Maybe we should put the Invite link to the BK RPG now or wait for the public release?